tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1533,6 +1533,8 @@ def sample_distorted_bounding_box(image_size, bounding_boxes, seed=None,
       Provide as input to `tf.image.draw_bounding_boxes`.
   """
   with ops.name_scope(name, 'sample_distorted_bounding_box'):
+    if min_object_covered is None:
```

Comment:
You should update the default value instead, although that will require an API review, I believe.

Reply:
The documentation already stated that the default is 0.1, so I think it would be fine to change it.